packages/ui/src/2_molecules/Table/components/TableDesktop/components/TableRow/TableRow.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
This is a 1:1 copy of `TableRow` from `TableBase` and I consider it very lazy. Delete one of those files and use the same file for both types of tables (have it in some shared location).